lib/client/autoform-map.coffee

```diff
@@ -60,7 +62,7 @@ initTemplateAndGoogleMaps = ->
 	@data.map = new google.maps.Map @find('.js-map'), mapOptions
 
 	if @data.value
-		location = if typeof @data.value == 'string' then @data.value.split ',' else [@data.value.lat, @data.value.lng]
+		location = if typeof @data.value == 'string' then @data.value.split ',' else if typeof @data.value == 'Array' then [@data.value.lat, @data.value.lng] else [@data.value[1],@data.value[0]]
```

Comment:
Is this ok? Seems like it's a bug there for me.

`if typeof @data.value == 'Array' then [@data.value.lat, @data.value.lng] else [@data.value[1],@data.value[0]]`